bedrock/firefox/redirects.py

```diff
@@ -557,4 +557,5 @@ def firefox_channel(*args, **kwargs):
     redirect(r"^firefox/sync/?$", "firefox.features.sync"),
     # issue 13732
     redirect(r"^firefox/welcome/3/?$", "firefox.accounts"),
+    redirect(r"^firefox/mobile/get-app/", "https://www.mozilla.org/en-US/firefox/browsers/mobile/get-app/"),
```

Comment:
```suggestion
    redirect(r"^firefox/mobile/get-app/", "firefox.browsers.mobile.get-app"),
```

Reply:
Can you please also add a test for this redirect at the bottom of the following file?

https://github.com/mozilla/bedrock/blob/main/tests/redirects/map_globalconf.py